ReportUtilityBills/README.md.erb

```diff
@@ -0,0 +1,41 @@
+<%#= README.md.erb is used to auto-generate README.md. %>
+###### (Automatically generated documentation)
+
+# <%= name %>
+
+## Description
+<%= description %>
+
+<%= modelerDescription %>
+
+## Arguments
+
+<% arguments.each do |argument| %>
+**<%= argument[:display_name] %>**
+
+<%= argument[:description] %>
+
+- **Name:** ``<%= argument[:name] %>``
+- **Type:** ``<%= argument[:type] %>``
+<% if argument[:units] %>
+- **Units:** ``<%= argument[:units] %>``
+<% end %>
+- **Required:** ``<%= argument[:required] %>``
+<% if argument[:type] == "Choice" %>
+- **Choices:** `<%= argument[:choice_values].join("`, `") %>`
+<% end %>
+<br/>
+<% end %>
+
+<% if arguments.size == 0 %>
+<%= "This measure does not have any user arguments" %>
+<% end %>
+
+<% if outputs.size > 0 %>
+## Outputs
+Measure outputs are:
```

Comment:
This makes me think: should we have an `outputs` method for this measure?

Reply:
Hmm, I looked at implementing it but I don't think we actually can. All of the outputs are [specific to utility bill scenarios](https://github.com/NREL/OpenStudio-HPXML/blob/dd8a2ce30246ac57e44f21a08113f75d60c5298f/ReportUtilityBills/measure.rb#L333-L340).

Reply:
So, similarly, emissions outputs for ReportSimOutput are excluded from the `outputs` method?

Why is B true?

Reply:
They wouldn't be additional, they'd be changing. We already register outputs with the runner, so these names would change. The only thing we don't do is populate them in the outputs section of the measure. So, for example, you can today in PAT retrieve the utility bill outputs but you have to type them in, whereas if the outputs section is available, you could choose from a dropdown menu.

Reply:
Hmm I'm confused then. Outputs in the `outputs` method couldn't be dynamically populated in the same way they are for the runner?

Reply:
I'll create an issue around this, but I don't want to tackle it as part of this PR.

Reply:
https://github.com/NREL/OpenStudio-HPXML/issues/1522